src/vs/workbench/contrib/chat/browser/chatWidget.ts

```diff
@@ -1022,13 +1023,9 @@ export class ChatWidget extends Disposable implements IChatWidget {
 					}, 0);
 				}
 			}));
-			const listEl = dom.append(container, $('.chat-welcome-history-list'));
+			const welcomeHistoryList = dom.append(container, $('.chat-welcome-history-list'));
```

Comment:
Why isn't this stuff in `ChatViewWelcomePart`? It's essentially part of the welcome view right?

Reply:
welcomeHistoryList is a bit misleading, it's shown on every empty state. i'll rename this